packages/libs/sdk/spec/api/graphApiClient/query.spec.ts

```diff
@@ -113,9 +101,10 @@ describe('graphApiClient.query', () => {
           }
         `;
 
-        await expect(
-          async () => await api.graphApiClient.query({ query })
-        ).rejects.toThrowError();
+        const { error } = await api.graphApiClient.query(query, {});
+        expect(error?.toString()).toMatch(
+          /cannot query field 'notafieldshoulderror' on type 'Collection'/
+        );
```

Comment:
With urql, invalid fields no longer error, they throw a warning and return the "error" field populated (and `data` is null in this case). More info in [the link in the warning](https://formidable.com/open-source/urql/docs/graphcache/errors/#4). 

Do we want this behavior or do we want to have it actually throw an error?

![image](https://github.com/quiknode-labs/qn-oss/assets/5964462/3fc83bb6-cc98-49f5-a416-31341a6527a4)